pkg/dnsserver/dnscontroller/lighthouse.go

```diff
@@ -0,0 +1,5 @@
+package dnscontroller
```

Comment:
I don't think there's any value with this being in its own file, ie just put it in handler.go.